packages/@expo/cli/src/export/embed/guessHermes.ts

```diff
@@ -0,0 +1,44 @@
+import fs from 'fs';
+import path from 'path';
+
+const ASSUME_HERMES_IS_USED_BY_DEFAULT = true;
```

Comment:
Oh thank you for that. I had expo-updates on a bare react-native 0.75 app and now the hermes_enabled line is removed in the podfile so it didn't detect that hermes is now enabled by default